web/modules/custom/dpl_redia_legacy/src/RediaEvent.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Drupal\dpl_redia_legacy;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Datetime\DrupalDateTime;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\dpl_event\EventWrapper;
+use Drupal\node\NodeInterface;
+use Drupal\recurring_events\Entity\EventInstance;
+
+/**
+ * An event object, containing the properties the RSS feed needs.
+ */
+class RediaEvent extends ControllerBase {
```

Comment:
Please reconsider if this class and `RediaEventMedia` needs to extend `ControllerBase`.

I do not understand why this is necessary. As I see it the class is not a controller and it does not utilize any methods provided by the parent class.

Reply:
They both utilize EntityTypeManager, that comes from that :)